storage/src/tests/common/testhelper.cpp

```diff
@@ -59,15 +59,13 @@ vdstestlib::DirConfig getStandardConfig(bool storagenode, const std::string & ro
     config = &dc.addConfig("stor-communicationmanager");
     config->set("rpcport", "0");
     config->set("mbusport", "0");
-    config = &dc.addConfig("stor-bucketdb");
     config->set("chunklevel", "0");
```

Comment:
This line can probably be removed as well, since it referred to `stor-bucketdb`

Reply:
Thanks,fixed